.ci/scripts/update-beats.sh

```diff
@@ -10,8 +10,8 @@ COMMIT_MESSAGE="Update to elastic/beats@$(go list -m -f {{.Version}} github.com/
 
 git checkout -b "update-beats-$(date "+%Y%m%d%H%M%S")"
 git add --ignore-errors go.mod go.sum NOTICE.txt \
-	.go-version docs/version.asciidoc \
-	docs/fields.asciidoc include/fields.go x-pack/apm-server/include/fields.go
+	.go-version docs/legacy/version.asciidoc \
+	docs/legacy/fields.asciidoc include/fields.go x-pack/apm-server/include/fields.go
```

Comment:
```suggestion
	.go-version docs/version.asciidoc
```

```suggestion
	.go-version docs/version.asciidoc \
```

- version.asciidoc is still in docs, not docs/legacy
- fields.asciidoc _is_ in docs/legacy, but we're no longer updating it or generating include/fields.go. The integration package is the way forward, and apm-server no longer installs index templates, etc.